tests/sqlparser_postgres.rs

```diff
@@ -5195,3 +5195,8 @@ fn parse_bitstring_literal() {
         ))]
     );
 }
+
+#[test]
+fn parse_select_without_projection() {
```

Comment:
if we use a dialect method suggesed above we'd likely want to move this to common and be able to write the  test using [all_dialects_where()](https://github.com/apache/datafusion-sqlparser-rs/blob/bcdc790b0135dc64ec4d40eaa837316530a5f5f3/tests/sqlparser_common.rs#L4981)